e2e/evm-evm/example/cfg/config_evm-evm.json

```diff
@@ -0,0 +1,33 @@
+{
+  "relayer": {},
+  "chains": [
+    {
+      "id": 1,
+      "name": "evm1",
+      "type": "evm",
+      "endpoint": "ws://evm1-1:8546",
+      "from": "0xff93B45308FD417dF303D6515aB04D9e89a750Ca",
```

Comment:
looks like from is not used actually, lets remove it from example

Reply:
from is used to pick private key from keystore

Reply:
oh sorry, misread
Will remove it from example.